### PR TITLE
[n/a] add Class List controller and documentation

### PR DESCRIPTION
**Pull request by gregkohn**
🚀  https://code.viget.com/stimulus-controllers/class-list/

### CODE REVIEW

**Review by davist11 — APPROVED** [review]

File: src/pages/class-list.astro [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            </Button>
+          </div>
+        </div>
+      </div>
+
+      <div slot="code">
+        <Markdown>
+          ```html
+          <div data-controller="class-list">
+            <p data-class-list-target="receiver">
```
Comment (joshuapease) [inline src/pages/class-list.astro]: Thought of a potential future enhancement (if it's even a good idea).

The ability to add something like `data-class-list-class-param` to various the `receiver` elements so you could add / remove / toggle different classes on different receivers. 

But maybe there isn't a compelling use case for it.

**Review by joshuapease — APPROVED** [review]
Tight tight tight

**Review by jeremyfrank — APPROVED** [review]

File: src/pages/class-list.astro (hunk 1 above)
Comment (gregkohn) [reply src/pages/class-list.astro]: ahh. i see. yeah i was thinking diff receiver behavior would be multiple controllers but that might be too constraining given markup. i think this is a good idea, lemme ponder that

Comment (joshuapease) [reply src/pages/class-list.astro]: Yeah... I think I'm still unsure of the best way to use composition with lots of tiny controllers.

Comment (gregkohn) [reply src/pages/class-list.astro]: noted here https://github.com/vigetlabs/stimulus-controllers/issues/14

**Comment by gregkohn** [comment]
(merged, not closed, just did the branch pushes in the wrong order)

**Pull request closed without merging**